compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeWindow.uikit.kt

```diff
@@ -129,11 +134,22 @@ internal actual class ComposeWindow : UIViewController {
             selectionWillChange = { skikoUIView.selectionWillChange() },
             selectionDidChange = { skikoUIView.selectionDidChange() },
         )
+        val uiKitPlatform = object : Platform by Platform.Empty {
+            override val textInputService: PlatformTextInputService = uiKitTextInputService
+            override fun viewConfiguration(densityProvider: () -> Density) =
+                object : ViewConfiguration {
+                    override val longPressTimeoutMillis: Long get() = 500
+                    override val doubleTapTimeoutMillis: Long get() = 300
+                    override val doubleTapMinTimeMillis: Long get() = 40
+                    override val touchSlop: Float
+                        get() = with(densityProvider()) { 3.dp.toPx() }
```

Comment:
is `3.dp` dictacted by some uikit guidelines or uikit source code?

Reply:
Just from my own experience